extras/ome_n3lo/large_n_limit.py

```diff
@@ -0,0 +1,38 @@
+"""This file contains the large-N limit of the diagonal Matrix elements.
+
+The expansions are obtained using the notebook Agg_Aqq_largex_expansion.nb.
+
+We note that:
+    * the limit og :math:`A_{qq}` is the same for non-singlet like and singlet-like expansions.
+    I.e. the local and singular part are the same
+    * the :math:`A_{qq,ps}` temr is vanishing in the large-x limit, i.e. it's only regular.
+"""
```

Comment:
```suggestion

In the following we make use of a small trick, which is available due to complex continuation. Although we are interested in the large-N limit, we can obtain that by observing the small-N limit. It is :math:`S_1(0)=0` (by continuation), :math:`S_1(1)=1` and :math:`S_1(N\to\infty) = \ln(N) - \gamma_E`.

"""
```
It took me 10 min to realize that you can indeed do that - but I believe you messed up the local part, because the $S_1$ contributes to the local part:
https://github.com/NNPDF/eko/blob/ee8f53310c27689dd345dfedd9cdc5e04fae9bfb/src/ekore/harmonics/w1.py#L33
and `Series[PolyGamma[0, n + 1] + EulerGamma, {n, \[Infinity], 0}]` yields `(EulerGamma-Log[1/n])+O[1/n]^1`

you can keep the "trick" or just return a tuple `(s1_coeff, explicit_const)` and recombine that in the suitable places

Reply:
Agreed, I'll provide them a mathematica file with the coefficients. 
Thanks for double checking.